cves/kernel/CVE-2022-28796.yml

```diff
@@ -55,7 +55,13 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
this is good! I think that you might want to also add something defining a race condition as well because its big jargon.

Reply:
I feel like the 'race condition' is a somewhat well known term among developers, especially given that it's not related to any within linux. I will add a little description however.